app/models/concerns/pick_up_order.rb

```diff
@@ -0,0 +1,89 @@
+class PickUpOrder < ActiveRecord::Base
+  # rubocop:disable Metrics/BlockLength
```

Comment:
I would agree re: the effort, which is why I used the comment to disable :wink:

I too agree that a config repo is a good idea.